Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py

```diff
@@ -611,7 +617,7 @@ def fetch_incidents():
     LOG('Fetching incidents since {}'.format(timestamp))
     netwitness_incidents = get_all_incidents(
         since=timestamp,
-        limit=100
+        limit=FETCH_LIMIT
```

Comment:
Not sure that the limit should be here, it should be in the incident creation side, we can miss incidents this way

Reply:
I'm not sure I follow. We have a limit on how many incidents can be pulled in at a time when `fetch_incidents` runs. It's user configurable in the integration settings.